doc/classes/AnimatedSprite.xml

```diff
@@ -34,6 +34,13 @@
 				Stops the current animation (does not reset the frame counter).
 			</description>
 		</method>
+		<method name="is_playing_backwards">
+			<return type="bool">
+			</return>
+			<description>
+				Wether or not the animation is currently playing backwards. This is set when calling [code]play()[/code].
```

Comment:
Typo: W**h**ether.
Also use `[method play]` to refer to the play method with a link.

Reply:
We also prefer to say `Returns [code]true[/code] if …, [code]false[/code] otherwise.` instead of `Whether …`, per the [Doc writing guidelines](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html).